Makefile

```
@@ -2,7 +2,7 @@
 
 GO_FILES=$(wildcard */*.go)
 BINARY=terraform-provider-propel
-VERSION=1.0.1
+VERSION=1.0.2
```

I think we should follow semantic versioning here. Since we are adding a feature in a backward compatible way, we should bump the minor version, right?
```suggestion
VERSION=1.1.0
```